obspy/io/sac/core.py

```diff
@@ -445,7 +445,7 @@ def _write_sac(stream, filename, byteorder="<", **kwargs):  # @UnusedVariable
         raise ValueError("Cannot open '%s'." % filename)
 
 
-def _internal_write_sac(trace, buf, byteorder="<", **kwargs):
+def _internal_write_sac(trace, buf, byteorder="<", keep_sac_header=True):
```

Comment:
Looks like `**kwargs` isn't used anywhere, since all tests still pass, so it might be safe to remove it, but maybe you can just be safe and keep `**kwargs`.